src/ParseServer.js

```diff
@@ -109,6 +113,37 @@ class ParseServer {
     return this._app;
   }
 
+  async getSecurityChecks() {
+    const config = Config.get(this.config.appId);
+    const response = await securityChecks(config);
+    const logger = logging.getLogger();
+    const warnings = response.response;
+    const clp = warnings.CLP;
+    const total = warnings.Total;
+    if (total == 0) {
+      return;
+    }
+    let errorString = `We found ${total} improvement${
```

Comment:
Please remove the pronouns "we", "you".
To keep the terminology consistent, I suggest we use `security warnings` across this whole PR.
To simplify code, I suggest to replace the singular/plural logic with `warning(s)`.
To keep the logs concise, please rewrite to something shorter, such as:

`${total} security warning(s) for Parse Server:\n\n`.